dev-util/clinfo/clinfo-2.1.17.02.09.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Generic tool to print information about all available OpenCL platforms and devices"
+HOMEPAGE="https://github.com/Oblomov/clinfo"
+SRC_URI="https://github.com/Oblomov/clinfo/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="CC0-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
```

Comment:
It definitely includes OpenCL headers, and probably links to the libraries. So it needs build deps too.